openslides/agenda/static/templates/agenda/slide-current-list-of-speakers-overlay.html

```diff
@@ -10,8 +10,10 @@ <h3 translate>Current list of speakers</h3>
     class="currentSpeaker">
     <i class="fa fa-microphone fa-lg"></i> {{ speaker.user.get_full_name() }}
     <ol class="nextSpeakers">
-      <li ng-repeat="speaker in agendaItem.speakers | filter: {begin_time: null} | orderBy:'weight' | limitTo: 3">
+      <li ng-repeat="speaker in nextSpeakers = (agendaItem.speakers | filter: {begin_time: null}) | orderBy:'weight' | limitTo: 3">
         {{ speaker.user.get_full_name() }}
     </ol>
+      <p ng-if="nextSpeakers.length > 3" class="lastSpeakers">
+       <i> {{ nextSpeakers.length - 3 }} <translate> more speakers</translate> </i>
```

Comment:
Suggestion: Use just this short string without translations:
`<i>+ {{ nextSpeakers.length - 3 }}</i>`